site/src/_layoutsPlayground/component.hbs

```diff
@@ -3,14 +3,15 @@
 <head>
     <meta charset="UTF-8">
     <title>{{default title 'd3fc'}}</title>
-    <link href="https://cdnjs.cloudflare.com/ajax/libs/d3fc/{{ package.version }}/d3fc.min.css" rel="stylesheet" />
+    <link href="/d3fc.css" rel="stylesheet" />
+    <!-- https://cdnjs.cloudflare.com/ajax/libs/d3fc/{{ package.version }}/ -->
 </head>
 <body>
     <svg id="chart" width="100%" height="300px"></svg>
 
     <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/{{ slice package.dependencies.d3 1 }}/d3.js"></script> -->
     <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/css-layout/{{ slice package.dependencies.css-layout 1 }}/css-layout.js"></script> -->
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3fc/{{ package.version }}/d3fc.bundle.min.js"></script>
+    <script src="/d3fc.bundle.js"></script>
```

Comment:
I'm always nervous about rooted paths, I can't think of a reason why it would matter at the minute but please can you change this to relative?